applet/libapplets/web_wifi/source/main.c

```diff
@@ -0,0 +1,63 @@
+// Include the most common headers from the C standard library
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <malloc.h>
+
+// Include the main libnx system header, for Switch development
+#include <switch.h>
+
+// See also libnx web.h.
+
+// This example shows how to use the WebWifiAuth LibraryApplet.
+
+// Main program entrypoint
+int main(int argc, char* argv[])
+{
+    Result rc=0;
+
+    // This example uses a text console, as a simple way to output text to the screen.
+    // If you want to write a software-rendered graphics application,
+    //   take a look at the graphics/simplegfx example, which uses the libnx gfx API instead.
```

Comment:
Comments should be [updated](https://github.com/switchbrew/switch-examples/blob/master/templates/application/source/main.c#L12):

```c
    // This example uses a text console, as a simple way to output text to the screen.
    // If you want to write a software-rendered graphics application,
    //   take a look at the graphics/simplegfx example, which uses the libnx Framebuffer API instead.
    // If on the other hand you want to write an OpenGL based application,
    //   take a look at the graphics/opengl set of examples, which uses EGL instead.
```